src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -205,6 +206,25 @@ public double getDefaultAlleleFrequency() {
     @Argument(fullName = MUTECT3_DATASET_LONG_NAME, optional = true, doc="Destination for Mutect3 data collection")
     public File mutect3Dataset;
 
+    @Advanced
+    @Argument(fullName = MUTECT3_DATASET_MODE_LONG_NAME, optional = true, doc="The type of Mutect3 dataset.  Depends on sequencing technology.")
+    public Mutect3DatasetMode mutect3DatasetMode = Mutect3DatasetMode.ILLUMINA;
+
+    public enum Mutect3DatasetMode {
+        ILLUMINA(11),
```

Comment:
I get a little nervous when you hardcode the sequencing technology, especially when given the same value.  Why not just make this a default?  What is the justification for this?  (We do not need to block this PR over this)

Reply:
Since Ultima uses unpaired reads it can't be encoded in the same way as Illumina data.  It's only a coincidence that the dimension of read tensors is 11 in both cases.

Reply:
(And note that this is an `Advanced` argument)